### ISSUE TITLE
Can't open files `Error: open out/[...] no such file or directory`

### ISSUE THREAD

**Issue by Tashima42** [enrichman/stegosecrets#17]
Buona sera! 
I'm trying to encrypt a file and it isn't working, even after building and using the binary, am I doing something wrong here?

```bash
pedro@Pedros-MacBook-Air stegosecrets % go run cmd/stego/stego.go encrypt --file ./README.md -p 5 -t 3
Encrypting './README.md'
Error: open out/./README.md.key: no such file or directory
```

**Comment by enrichman** [comment]
Ciao! Yes, the command is correct but it's assuming to have an existing `out` directory where to store the files.
This is a bug, we should at least create the folder if it doesn't exists.

**Comment by Tashima42** [comment]
What do you think about using something like this function to create the folder?

https://github.com/enrichman/stegosecrets/pull/20/files#diff-09669fc96c318226e77e7d95f799a0b3e1f8bbf99f4fbee778b1237a58373e9fR58-R63

**Comment by enrichman** [comment]
Fixed with #22
